ext/standard/string.c

```diff
@@ -2109,13 +2109,11 @@ PHP_FUNCTION(strripos)
 	needle_dup = zend_string_tolower(needle);
 	if ((found = (char *)zend_memnrstr(p, ZSTR_VAL(needle_dup), ZSTR_LEN(needle_dup), e))) {
```

Comment:
Follow-up PR/commit suggestion.
```suggestion
	if ((found = zend_memnrstr(p, ZSTR_VAL(needle_dup), ZSTR_LEN(needle_dup), e))) {
```

Reply:
Feel free to commit that ;)